package.json

```diff
@@ -42,6 +42,7 @@
     "requireindex": "^1.2.0"
   },
   "devDependencies": {
+    "@testing-library/dom": "^7.27.1",
```

Comment:
Hey @AntonNiklasson i just realized we should make those a dependency not Dev dependency since it's used directly.  After that I think we can merge